package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "github-ci-bot",
+  "name": "kie-ci-bot",
```

Comment:
this shouldn't be in this PR but in https://github.com/kiegroup/github-ci-bot/pull/5

Reply:
Yeah, I sent this PR over that one so no merge conflict will happen. will remove this commit.